manifest_staging/charts/eraser/templates/eraser-manager-role-clusterrole.yaml

```diff
@@ -28,29 +16,6 @@ rules:
   - get
   - list
   - watch
-- apiGroups:
-  - ""
-  resources:
-  - pods
-  verbs:
-  - create
-  - delete
-  - get
-  - list
-  - update
-  - watch
-- apiGroups:
-  - ""
-  resources:
-  - podtemplates
-  verbs:
-  - create
-  - delete
-  - get
-  - list
-  - patch
-  - update
-  - watch
 - apiGroups:
   - eraser.sh
```

Comment:
For the eraser resources, does your controller need access to `create` and similar permissions on the resource itself (i.e. not `/status`)?

Reply:
It does for ImageJob, but not for ImageList. Removed create/delete/update/patch permissions.